src/core/java/app/com/linkedin/playparseq/j/PlayParSeq.java

```diff
@@ -38,6 +39,17 @@
    */
   <T> Task<T> toTask(final String name, final Callable<CompletionStage<T>> f);
 
+  /**
+   * The method toTask converts a {@code Callable<CompletionStage<T>>} to a ParSeq {@code Task<T>}.
+   *
+   * @param name The String which describes the Task and shows up in a trace
+   * @param executionContext The Executor of the async CompletionStage
+   * @param f The Callable which returns a CompletionStage
+   * @param <T> The type parameter of the CompletionStage and the ParSeq Task
+   * @return The ParSeq Task
+   */
+  <T> Task<T> toTask(final String name, final HttpExecutionContext executionContext, final Callable<CompletionStage<T>> f);
```

Comment:
To match CompletionStage's API style, I suggest moving `HttpExecutionContext` to the last parameter. Same elsewhere.

Reply:
Got it. Will do.